src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+OS=`cat /etc/os-release | grep "^ID=" | sed 's/ID=//g' | sed 's/["]//g' | awk '{print $1}'`
+echo "Operating System: ${OS}"
+
+realm="TEST.COREFX.NET"
+
+principal1="HOST/host.test.corefx.net"
+principal2="HTTP"
+krb_user="krb_user"
+password="password"
```

Comment:
If someone runs this script, is there any danger that it'll open their machine up to any kind of attack? e.g. could it give someone with these fake creds access to anything on their machine?